src/functions-and-arrays.js

```diff
@@ -52,14 +133,34 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(list) {
+  let listCopy = []
+  if(list.length === 0){
+    return null
+  }else {
+    for (let i = 0; i < list.length; i++) {
+      if (!listCopy.includes(list[i])) {
+        listCopy.push(list[i])
+      }
+    }
+    return listCopy
+  }
+  
```

Comment:
Bien resuelto. Nada más comenzar no se espera que encontréis una solución compleja a esta kata pero la forma más elegante (puedes investigarlo) de resolver este problema sería a través de un Set: https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Set

Un Set es una estructura de datos que alberga elementos ÚNICOS no repetidos.